src/sdk/eng/Signing.props

```diff
@@ -84,6 +84,14 @@
   
   <ItemGroup>
     <FileSignInfo Include="StreamJsonRpc.dll" CertificateName="MicrosoftSHA2" />
+
+    <!-- Roslyn apphosts -->
+    <FileSignInfo Include="csc" CertificateName="$(DotNetCertificateName)" />
+    <FileSignInfo Include="vbc" CertificateName="$(DotNetCertificateName)" />
+    <FileSignInfo Include="VBCSCompiler" CertificateName="$(DotNetCertificateName)" />
```

Comment:
Mac needs a different cert: https://github.com/dotnet/runtime/blob/0e3562e97c6db531f26a2ffe3e8084cf67ba8a93/eng/Signing.props#L42-L46

you can just update this one since we don't sign Linux executables

Reply:
hm I think it'd make more sense to move this to Roslyn's Signing.props